test/libsolidity/SolidityNatspecJSON.cpp

```diff
@@ -204,6 +204,112 @@ BOOST_AUTO_TEST_CASE(dev_and_user_no_doc)
 	checkNatspec(sourceCode, "test", userNatspec, true);
 }
 
+BOOST_AUTO_TEST_CASE(dev_public_state_variable)
```

Comment:
Actually I think this whole testing framework is a bit weird.

It may be better to just use one or two commandline tests instead.